tests/w-xtm-bridge.test.ts

```
@@ -4,57 +4,168 @@ import {
   beforeAll,
-  afterAll
-} from "matchstick-as/assembly/index"
-import { Address, BigInt } from "@graphprotocol/graph-ts"
-import { OwnershipTransferred } from "../generated/schema"
-import { OwnershipTransferred as OwnershipTransferredEvent } from "../generated/wXTMBridge/wXTMBridge"
-import { handleOwnershipTransferred } from "../src/w-xtm-bridge"
-import { createOwnershipTransferredEvent } from "./w-xtm-bridge-utils"
+  afterAll,
+  logStore,
+} from "matchstick-as/assembly/index";
+import { Address, BigInt, Bytes } from "@graphprotocol/graph-ts";
```

Biome (1.9.4)

[error] Do not shadow the global "BigInt" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)